test/index.js

```diff
@@ -60,23 +60,38 @@ describe('Schwifty', () => {
         useNullAsDefault: true
     };
 
-    const getServer = (options, cb) => {
+    async function getServer (options) {
 
         const server = new Hapi.Server();
-        server.connection();
 
-        server.register({
-            register: Schwifty,
-            options
-        }, (err) => {
+        try {
 
-            if (err) {
-                return cb(err);
-            }
+            await server.register({
+                plugin: Schwifty,
+                options
+            });
 
-            return cb(null, server);
-        });
-    };
+            return server;
+        } catch (err) {
+
+            console.log("Initial registration failed", err);
+            throw err;
+        }
+
+    }
+
+    async function initializeServer (server) {
+
+        try {
+
+            await server.initialize();
+            return server;
+        } catch (error) {
+
+            console.log("Initialization error", error);
```

Comment:
I assume this logging is the main reason for wrapping `server.initialize()` in a function– yes?

Reply:
oh man...naw, the main reason is because I just followed the examples I saw on the release notes as I was struggling to get this stuff to click in my brain :) 

I think I see your point, though. Super helpful note, thanks, dude 🙏